docs/design/Pre-UpgradeAnalysis.md

```diff
@@ -77,9 +77,9 @@ Detailed instructions on how to set up the decentralized solution will be provid
 
 ## Phase III:
 Add loose assembly analysis capability to upgrade-assistant.
-- Chem integration.
+- Loose assembly analysis integration.
 
-**Cost** : L (This is L coz this requires quite a bit of experimental prototyping to know the best way to integrate Chem. This effort can happen in parallel to Phase I and Phase II)
+**Cost** : L (This is L coz this requires quite a bit of experimental prototyping to know the best way to integrate loose assembly analysis. This effort can happen in parallel to Phase I and Phase II)
```

Comment:
'coz'? :)

Reply:
rewording because it doesn't speak to the feature but rather an implementation detail.